tools/ctl/ctl.go

```diff
@@ -462,6 +467,68 @@ var listRuns = &cobra.Command{
 	},
 }
 
+var runGCB = &cobra.Command{
+	Use:   "run-gcb -project <ID> --ecosystem <ecosystem> --package <name> --version <version> [--artifact <name>]",
+	Short: "Trigger a GCB build",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		r := debian.Rebuilder{}
+		mux := rebuild.RegistryMux{
+			Debian:   debianreg.HTTPRegistry{Client: http.DefaultClient},
+			CratesIO: cratesioreg.HTTPRegistry{Client: http.DefaultClient},
+			NPM:      npmreg.HTTPRegistry{Client: http.DefaultClient},
+			PyPI:     pypireg.HTTPRegistry{Client: http.DefaultClient},
+		}
+		t := rebuild.Target{
+			Ecosystem: rebuild.Ecosystem(*ecosystem),
+			Package:   *pkg,
+			Version:   *version,
+			Artifact:  *artifact,
+		}
+		s, err := r.InferStrategy(cmd.Context(), t, mux, nil, nil)
+		if err != nil {
+			log.Fatal(err)
+		}
+		log.Println("Using strategy: ", s)
+		dockerfile, err := rebuild.MakeDockerfile(rebuild.Input{
+			Strategy: s,
+			Target:   t,
+		}, rebuild.RemoteOptions{})
+		if err != nil {
+			log.Fatal(err)
+		}
+		log.Println("Dockerfile: ", dockerfile)
+		dir, err := os.MkdirTemp("/tmp", "")
+		if err != nil {
+			log.Fatal(err)
+		}
+		f, err := os.Create(filepath.Join(dir, "dockerfile"))
```

Comment:
you should be able to pipe the dockerfile content into the docker build command by specifying the docker context (last argument) as "-"